code/modules/reagents/reagent_containers.dm

```diff
@@ -156,6 +156,16 @@
 	if(beaker_weakness_bitflag & PH_WEAK)
 		if((reagents.pH < 1.5) || (reagents.pH > 12.5))
 			START_PROCESSING(SSobj, src)
+	else if((reagents.pH < -3) || (reagents.pH > 17))

```

Comment:
Just to point out, this `else if` means beaker susceptible to akalyne or acid Ph are still allowed to exceed acceptable values. No great impact to the game however, cause those containers will be destroyed in seconds anyway. Your call.

Reply:
Mm! I'm fine with it working like that because the flagged containers have melting sprites.